bots/discord/ree6/config.yml

```diff
@@ -40,7 +40,7 @@ bot:
   misc:
     status: ree6.de | %guilds% Servers. (%shard%) # The Status of the Bot.
     feedbackChannelId: 0 # The Channel used for Feedback.
-    ownerId: 321580743488831490 # The ID of the Bot Owner. Change this to yours!
+    ownerId: <YOUR DISCORD ID> # The ID of the Bot Owner. Change this to yours!
```

Comment:
Good catch! Although, I would change it to `YOUR_DISCORD_ID` just to avoid using special characters in YAML (even though it probably still works as is).